pkg/platform/model/vcs.go

```diff
@@ -204,21 +221,24 @@ func NewNamespacePlatform() Namespace {
 	return Namespace{NamespacePlatform, "platform"}
 }
 
-const OrgNamespacePrefix = "private/"
-
 func NewOrgNamespace(orgName string) Namespace {
```

Comment:
Nit: for consistency we should rename this `NewNamespaceOrg()` to match the other functions.